mnefun/_mnefun.py

```diff
@@ -2252,7 +2254,7 @@ def do_preprocessing_combined(p, subjects, run_indices):
                                  tmin=eog_t_lims[0], tmax=eog_t_lims[1],
                                  l_freq=None, h_freq=None, no_proj=True,
                                  ch_name=p.eog_channel,
-                                 reject=p.ssp_eog_reject)
+                                 reject=p.ssp_eog_reject, average=p.proj_ave)
```

Comment:
So `compute_proj_exg` now computes the projections based on the averaged artifact epochs? What was it doing before?

Reply:
Computing them based on the covariance of the epochs (which is slightly different). In theory if the events are time-locked, which ECG/EOG events appear to be for us, then I think computing it based on the Evoked should work better

Reply:
Okay. Yeah, I am pretty sure that's how it's benn done in BESA for ever, I just assumed it was done based on averaging. Good to know, I'll try it.